shell/platform/linux/fl_application.cc

```diff
@@ -0,0 +1,183 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_application.h"
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_dart_project.h"
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_plugin_registry.h"
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_view.h"
```

Comment:
nit: keep the first `#include` where it is, move `fl_dart_project.h` through `fl_view.h` down below the GDK `#include`.

Order is:
1. Associated header
2. C includes
3. C++ includes
4. Other includes from within flutter project. For whatever reason we also tend to dump gtest in here.